src/styles/utils/variables.scss

```diff
@@ -0,0 +1 @@
+$button-color: #00acdc;
```

Comment:
don't be afraid to make more variables)))  the more - the easier to modify.

Reply:
Okay. I will try to use it in the next projects ;)